src/DurableTask.AzureStorage/AzureStorageOrchestrationService.cs

```diff
@@ -1890,6 +1892,28 @@ public void Dispose()
             this.orchestrationSessionManager.Dispose();
         }
 
+        /// <summary>
+        /// Gets the status of all orchestration instances with paging that match the specified conditions.
+        /// </summary>
+        public Task GetOrchestrationStateWithFiltersAsync(OrchestrationStatusQueryCondition condition, CancellationToken cancellationToken)
+        {
+            var transferredCondition = TransferToAzureStorageCondition(condition);
```

Comment:
Let's not use `var` here since we can't immediately see the variable type.
```suggestion
            OrchestrationInstanceStatusQueryCondition convertedCondition = TransferToAzureStorageCondition(condition);
```